src/components/PopupWithForm.js

```diff
@@ -16,15 +17,19 @@ export default class PopupWithForm extends Popup {
     return formValues;
   }
 
-  // closePopup() {
-  //   super.closePopup();
-  // }
+ openPopup() {
+  this._saveButton.textContent = "Save";
+  super.openPopup();
+ }
 
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._popupForm
  }


```